### PR TITLE
Move bundle contents update endpoint to /bundles

### PR DESCRIPTION
**Pull request by kashizui**
Leaving further modifications to the upload method for future pull requests, but this should clear the worker system for deployment before the rest of the REST API.

@klopyrev

### CODE REVIEW

**Comment by kashizui** [comment]
@klopyrev Update: successfully retested worker system end-to-end.

File: codalab/rest/bundle.py [hunk 1]
```diff
@@ -61,6 +65,40 @@ def get_blob(uuid, path=''):
...
+    Update the contents of the given running or uploading bundle.
+
+    Accepts the filename as a query parameter, used to determine whether the
+    upload contains an archive.
+
+    Bundle state is updated to READY after upload is complete if the |finalize|
+    query parmater is nonzero.
+    """
+    bundle = local.model.get_bundle(uuid)
+    if not check_bundle_have_run_permission(local.model, request.user.user_id, bundle):
```
Comment (klopyrev) [inline codalab/rest/bundle.py]: Change to 

```
check_bundles_have_all_permission(local.model, request.user, [uuid])
bundle = local.model.get_bundle(uuid)
```

run permission is a bit simpler and really reserved for the worker.

File: codalab/rest/bundle.py [hunk 2]
```diff
@@ -61,6 +65,40 @@ def get_blob(uuid, path=''):
...
+@put('/bundles/<uuid:re:%s>/contents/blob/' % spec_util.UUID_STR,
+     apply=AuthenticatedPlugin())
+def update_blob(uuid):
+    """
+    Update the contents of the given running or uploading bundle.
+
+    Accepts the filename as a query parameter, used to determine whether the
+    upload contains an archive.
+
+    Bundle state is updated to READY after upload is complete if the |finalize|
```
Comment (klopyrev) [inline codalab/rest/bundle.py]: Not implemented. When you do implement, change to FAILED if there is an exception during upload.

**Comment by klopyrev** [comment]
LGTM

**Comment by klopyrev** [comment]
Make sure you address the comment.

File: codalab/rest/bundle.py (hunk 2 above)
Comment (kashizui) [inline codalab/rest/bundle.py]: Will do.

**Pull request merged**